build/Dockerfile

```diff
@@ -29,9 +29,9 @@
 
 # Declare the BASE_IMAGE argument in the first line, for more detail
 # see: https://github.com/moby/moby/issues/38379
-ARG BASE_IMAGE=openjdk:8-jdk
+ARG BASE_IMAGE=eclipse-temurin:8-jdk-focal
 
-FROM maven:3.6-jdk-8 as builder
+FROM eclipse-temurin:8-jdk-focal as builder
```

Comment:
it's hard to maintain the maven version in multi places, so it's better to use `build/mvn` to auto-download the expected version.